multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -51,14 +51,34 @@ def multilingual():
 
 
 @automm_presets.register()
-def zero_shot():
+def zero_shot_classification():
```

Comment:
Does it make sense to use `zero_shot_classification ` only for text?

Reply:
In hf this pipeline is NLP only.